packages/salesforcedx-vscode-core/src/commands/util/retrieveParser.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
```

Comment:
Decided to have the parser as a function vs a class since there was not a lot of benefits from using a class here.